src/components/forms/AddServiceForm.js

```diff
@@ -18,6 +26,7 @@ function AddServiceForm({
   serviceTypes,
 }) {
   const [form] = Form.useForm();
+  // console.log("serviceForm", state)
```

Comment:
Would say we could remove the console.log here

Reply:
got it